pxd_core.h

```diff
@@ -36,6 +39,7 @@ struct pxd_device {
 	struct pxd_context *ctx;
 	bool connected;
 	mode_t mode;
+	bool fastpath;
```

Comment:
Why is fastpath per device property? Are we planning to allow mixed set of devices?

Reply:
Ofcourse it is per device. I believe we had a discussion on this yesterday. Devices shall be attached in fastpath only upon request from px and still can always dynamically switch to slowpath if there are any kind of IO errors as a fallback.